tests/mocks/githubrest.py

```diff
@@ -195,6 +195,6 @@ def get_repos_for_user(
     def get_enterprise_audit_logs(self, **kwargs: Any):
         url = (
             f"{self.base_url}/enterprises/test-enterprise"
-            f"/audit-log?per_page=100&phrase=action:protected_branch.create"
+            f"/audit-log?per_page=100&phrase=%20action%3Aprotected_branch.create+actor%3Atest-include-actor+-actor%3Atest-exclude-actor"
         )
         self.add_response(url=url, **kwargs)
```

Comment:
To be useful as a test, you should leave the phrase section to the caller.
```suggestion
def get_enterprise_audit_logs(self, *, search_phrase: str, **kwargs: Any):
    url = (
        f"{self.base_url}/enterprises/test-enterprise"
        f"/audit-log?per_page=100&phrase={search_phrase}"
    )
    self.add_response(url=url, **kwargs)
```